src/core/components/accordion/accordionItemHeader/accordionItemHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+    AccordionHeader as RadixAccordionHeader,
+    AccordionTrigger as RadixAccordionTrigger,
+} from '@radix-ui/react-accordion';
+import classNames from 'classnames';
+import { forwardRef, type ComponentPropsWithRef } from 'react';
+import { AvatarIcon } from '../../avatars';
+import { IconType } from '../../icon';
+
+export const AccordionItemHeader = forwardRef<HTMLButtonElement, ComponentPropsWithRef<typeof RadixAccordionTrigger>>(
+    ({ children, className, ...otherProps }, forwardedRef) => (
```

Comment:
We can destructure the props object inside the component and rename `forwardedRef` to `ref` to keep the implementation aligned to all the other components and avoid having a double tab spacing. Same for all the other components
```suggestion
export const AccordionItemHeader = forwardRef<HTMLButtonElement, IAccordionItemHeaderProps>(
    (props, ref) => {
    const { children, className, ...otherProps } = props;
```